pkg/controlplane/instance.go

```diff
@@ -396,7 +396,7 @@ func (c CompletedConfig) StorageProviders(client *kubernetes.Clientset) ([]contr
 			NodePortRange:           c.Extra.ServiceNodePortRange,
 			IPRepairInterval:        c.Extra.RepairServicesInterval,
 		},
-	})
+	}, c.ControlPlane.Generic.Authorization.Authorizer)
```

Comment:
I think is better if we add a reference to the original issue https://github.com/kubernetes/kubernetes/issues/133515 as this is hard to understand without context, at least people can trace the origin without git blame